### PR TITLE
Add Networking_In_Java.yml

### PR DESCRIPTION
**Pull request by PrajjawalBanati**

### CODE REVIEW

File: _data/projects/Networking_In_Java.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        - socket-programming
+        - stream-programming
+        - multithreaded-server
+        - tcp-server
+        - udp-server
+        - java-programming
+upforgrabs:
+        name: help wanted
+        link: https://github.com/PrajjawalBanati/Networking_In_Java/labels/help%20wanted
+stats:
```
Comment (shiftkey) [inline _data/projects/Networking_In_Java.yml]: No need to add this part of the file at this time - once this is reviewed and merged it'll get generated based on the activity on GitHub.

Comment (PrajjawalBanati) [reply _data/projects/Networking_In_Java.yml]: Ok sir, Thank you so much.

**Comment by shiftkey** [comment]
@PrajjawalBanati you're so close to helping the bot give this PR a check! The current issue is that the label name is slightly off:

 - in this PR you've used `Up-for-Grabs`
 - but in the [list of labels](https://github.com/PrajjawalBanati/Networking_In_Java/labels/Up-For-Grabs) itself there's `Up-For-Grabs` (with a capital `F`)

If you make that change it should pass.

File: _data/projects/Networking_In_Java.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        - java
+        - java-networking
+        - socket-programming
+        - stream-programming
+        - multithreaded-server
+        - tcp-server
+        - udp-server
+        - java-programming
+upforgrabs:
+        name: Up-for-Grabs
```
Comment (shiftkey) [inline _data/projects/Networking_In_Java.yml]:
```suggestion
        name: Up-For-Grabs
```

File: _data/projects/Networking_In_Java.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        link: https://github.com/PrajjawalBanati/Networking_In_Java/labels/Up-for-Grabs
```
Comment (shiftkey) [inline _data/projects/Networking_In_Java.yml]:
```suggestion
        link: https://github.com/PrajjawalBanati/Networking_In_Java/labels/Up-For-Grabs 
```

**Comment by PrajjawalBanati** [comment]
Thank you, @shiftkey sir, for your help

**Review by shiftkey — APPROVED** [review]

**Pull request merged**